core/src/state.rs

```diff
@@ -618,9 +619,9 @@ impl<'world> WorldBlock<'world> {
     /// Commit block's changes
     pub fn commit(self) {
         // IMPORTANT!!! Commit fields in reverse order, this way consistent results are insured
+        self.executor_data_model.commit();
         self.executor.commit();
```

Comment:
```suggestion
        self.executor.commit();
```

why isn't executor data model inside executor?

Reply:
For the same reason why permissions schema was not inside executor. I just followed the original pattern.

Why should it be a single entry in the `World`?